prompt/prompt-executor/prompt-executor-clients/prompt-executor-openai-client/src/commonMain/kotlin/ai/koog/prompt/executor/clients/openai/OpenAIModels.kt

```diff
@@ -19,6 +19,7 @@ import ai.koog.prompt.llm.LLModel
  * | [Reasoning.O3Mini]               | Medium    | $1.1-$4.4          | Text, Tools                  | Text, Tools        |
  * | [Reasoning.O3]                   | Slowest   | $10-$40            | Text, Image, Tools, Document | Text, Tools        |
  * | [Reasoning.O1]                   | Slowest   | $15-$60            | Text, Image, Tools, Document | Text, Tools        |
+ * | [Reasoning.GPT5Pro]              | Slowest   | $15-$120           | Text, Image, Tools, Document | Text, Tools        |
```

Comment:
Input: Text, image
Output: Text
```suggestion
 * | [Reasoning.GPT5Pro]              | Slowest   | $15-$120           | Text, Image,                 | Text.              |
```